tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -1896,6 +1896,27 @@ protected function addMockConnection($model)
             return new BaseBuilder($connection, $grammar, $processor);
         });
     }
+
+    public function testTouchingModelWithTimestamps()
+    {
+        $this->assertFalse(
+            Model::isIgnoringTouch(Model::class)
+        );
+    }
+
+    public function testTouchingModelWithUpdatedAtNull()
+    {
+        $this->assertFalse(
```

Comment:
This case has to return `true`. You need to cover it by adjusting `isIgnoringTouch()`.

Reply:
done